bin/validate.js

```diff
@@ -342,7 +342,7 @@ vendors.vendors.forEach((v) => {
 
           if (!vendorProfiles[vendorID][regionProfile.id]) {
             const profile = yaml.load(fs.readFileSync(`./vendor/${vendorID}/${regionProfile.id}.yaml`));
-            if (profile.vendorProfileID) {
+            if ('vendorProfileID' in profile) {
```

Comment:
This is because the value `0` is falsy in Javascript. Your fix checks explicitly if the field is in the object which is better. 

The more idiomatic way if doing this is `if (typeof profile.vendorProfileID !== 'undefined')`. Technically, it is also okay if the value is undefined. However, that will never happen when reading from YAML, so it's only more idiomatic.

Reply:
Ahh i see, thanks! Will keep it in mind.